samples/bluetooth/direct_test_mode/CMakeLists.txt

```
@@ -5,11 +5,11 @@
 #
 cmake_minimum_required(VERSION 3.20.0)
 
-if (DEFINED SHIELD AND ${BOARD} STREQUAL "nrf5340dk_nrf5340_cpunet")
+if(DEFINED SHIELD AND "${BOARD}" STREQUAL "nrf5340dk_nrf5340_cpunet")
   string(REPLACE " " ";" SHIELD_LIST "${SHIELD}")
   foreach(sh IN_LIST SHIELD_LIST)
```

There is a typo here, this worked previously by coincident. Now with quotes it does not work because of this bug.
Could you fix it also?
```suggestion
  foreach(sh IN LISTS SHIELD_LIST)
```

---

`IN_LIST` is correct, as per https://cmake.org/cmake/help/v3.3/command/if.html

---

Fixed